carbon-design-system/carbon-addons-iot-react
# fix(FlyoutMenu): add new prop triggerClassname

Closes #3624 

**Summary**

- The `.bx--tooltip__label `has a blue outline with 0 pixels. It shows as a blue dot when tab focus on it.
- We need to expose triggerClassName to set the `outline:none` on the `.bx--tooltip__label` for the FlyoutMenu Tooltip. 

 
![image](https://user-images.githubusercontent.com/24279794/199313239-7cbd89a0-c615-4701-9e93-6cbe53fa7e06.png)
![image](https://user-images.githubusercontent.com/24279794/199313293-94010d0c-e6d4-46ba-9ea8-0fefa2f17e67.png)


**Change List (commits, features, bugs, etc)**

- add prop triggerClassname in FlyoutMenu
- set triggerClassname in DateTimePickerV2

**Acceptance Test (how to verify the PR)**

- go to single select or range select story
- click tab in keyboard
- see focus will not move to a blue dot label under the dataTime Input



**Things to look for during review**
